package.json

```diff
@@ -60,6 +60,7 @@
     "is-promise": "^2.1.0",
     "lodash": "^4.12.0",
     "lodash-es": "^4.12.0",
+    "promise": "^7.1.1",
```

Comment:
promise is already used by another module in the project so I thought it would be best to use it here too